### PR TITLE
Update streamers.csv with simply cyber YT

### PR DESCRIPTION
**Pull request by gerryguy311**
added simply cyber youtube link.

### CODE REVIEW

File: streamers.csv [hunk 1]
```diff
@@ -220,6 +220,7 @@ Sharghaas,
 shelldredd,
 shenetworks,https://www.youtube.com/c/shenetworks
 SherlockSec,
+Simply Cyber,https://www.youtube.com/@simplycyber
```
Comment (goproslowyo) [inline streamers.csv:223]: Hey @gerryguy311 thanks for the PR, I want to double-check your Twitch username before I merge!!

```suggestion
gerald_auger_simplycyber,https://www.youtube.com/@simplycyber
```

**Review by goproslowyo — CHANGES_REQUESTED** [review]

File: streamers.csv (hunk 1 above)
Comment (goproslowyo) [reply streamers.csv:223]: Oh btw, this won't show you online on Twitch (and not YT, unfortunately no API to query that info) unless you're streaming in the Just Chatting, Software & Game Dev, or Science & Technology categories.

Comment (goproslowyo) [reply streamers.csv:223]: If you mutli-stream from YT to Twitch, I think you'll be all good tho :).

**Comment by gerryguy311** [comment]
I can start streaming to Twitch in addition just don’t shows up. Lmk. I use
restream so it’s just a toggle.  Thx.

Also I think my twitch is Geraldaugersimplycyber

File: streamers.csv (hunk 1 above)
Comment (gerryguy311) [reply streamers.csv:223]: I will start pushing to twitch and YT so it does show up. https://www.twitch.tv/gerald_auger_simplycyber

Comment (gerryguy311) [reply streamers.csv:223]: I think im talk shows and podcasts catefogry, but if i can figure it out ill change it to Science and Tech so it does show up when im live.  I dont use twitch much so have to look around to see. 

Thank you so much.

**Pull request merged**